examples/plot_clickable_image.py

```diff
@@ -0,0 +1,65 @@
+"""
+=========================================
+Demonstration of how to use ClickableImage / generate_2d_layout.
+=========================================
+
+In this example, we open an image file, then use ClickableImage to
+return 2D locations of mouse clicks (or load a file already created).
+Then, we use generate_2d_layout to turn those xy positions into a layout
+for use with plotting topo maps. In this way, you can take arbitrary xy
+positions and turn them into a plottable layout.
+"""
+
+from scipy.ndimage import imread
+import numpy as np
+from matplotlib import pyplot as plt
+import mne
+from mne.viz.utils import ClickableImage
+from mne.channels.layout import generate_2d_layout
+
+# Set parameters and paths
+plt.rcParams['image.cmap'] = 'gray'
+im_path = '/Users/choldgraf/github/mne-python/mne/data/image/mni_brain.gif'
```

Comment:
I can't run this example currently -- this will need to be more like `op.path(op.dirname(__file__), '..', 'mne', 'data', 'image', 'mni_brain.gif')`